spec/minidoc/timestamps_spec.rb

```diff
@@ -20,5 +22,27 @@ class TimestampsUser < Minidoc
       user.save
       expect(user.created_at).to_not eq user.updated_at
     end
+
+    it "updates updated_at when doc is changed with #set" do
+      user = TimestampsUser.create
+      sleep 0.001
+      user.set(name: "Abby Normal")
+      expect(user.created_at).to_not eq user.updated_at
+    end
+
+    it "updates updated_at when doc is changed with .atomic_set" do
```

Comment:
Oh, I did change the spec title to accurately reflect what was being tested, though.